hw/ip/sram_ctrl/doc/dv/index.md

```diff
@@ -120,8 +139,8 @@ Some of the most commonly used tasks / functions are as follows:
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
 The following covergroups have been developed to prove that the test intent has been adequately met:
-* cg1:
-* cg2:
+* b2b_access_types_cg (imported from mem_bkdr_scb): Cover that all combinations of reads/writes can create b2b scenario
```

Comment:
Sorry, this description is not clear to me, do you mean back to back read and write requests to the same addresses? Also (nit), please add a closing period here and below.

Reply:
Updated, hope it is clear. Thanks